website/src/pages/formatter/index.mdx

```diff
@@ -9,8 +9,10 @@ import PackageManagerRomeCommand from "@src/components/PackageManagerRomeCommand
 
 # Formatter
 
-Biome's an opinionated formatter that has the goal to stop all ongoing debates over styles. It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html), only supporting
-few options to avoid that debates over styles turn into debates over Biome options. It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.
+Biome's an opinionated formatter that has the goal to stop all ongoing debates over styles.
+It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
+only supporting few options to avoid that debates over styles turn into debates over Biome options.
+It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.
```

Comment:
```suggestion
It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
only supporting a few options to avoid debates over styles, turning into debates over Biome options.
It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.
```